cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -1038,7 +1038,43 @@ def _restart_manager_db_dependent_services(self):
         common.sudo(['systemctl', 'restart', 'cloudify-amqp-postgres'])
         common.sudo(['systemctl', 'restart', 'cloudify-restservice'])
 
-    def add_cluster_node(self, address):
+    def _node_is_in_db(self, node_id):
```

Comment:
I would here just return the num with the same node_id, and move the logic of ==1 to the add command

Reply:
That would only make sense if this function weren't called '_node_is_in_db'. Is there a benefit to splitting this logic?